editor/src/clj/editor/asset_browser.clj

```diff
@@ -430,13 +430,15 @@
                 {:title "Delete Files?"
                  :icon :icon/circle-question
                  :header "Are you sure you want to delete these files?"
-                 :content {:fx/type fxui/label
+                 :content {:fx/type :v-box
                            :style-class "dialog-content-padding"
-                           :text (format "You are about to delete: \n%s"
-                                         (->> selection
-                                              (map #(str "\u00A0\u00A0\u2022\u00A0"
-                                                         (resource/resource-name %)))
-                                              (string/join "\n")))}
+                           :children [{:fx/type fxui/label
+                                       :text "You are about to delete:"}
+                                      {:fx/type fxui/text-area
```

Comment:
Btw isn't text-area editable by default? We probably don't want it to be editable. There is also (a private) `info-dialog-text-area` in dialogs ns (see [here](https://github.com/defold/defold/blob/3e5fda5b774763bf1390b32ad7246024a8a3dc25/editor/src/clj/editor/dialogs.clj#L164)). Maybe we want a different dialog there?

Reply:
i.e. `dialogs/make-info-dialog` could be a better fit here (it wraps confirmation dialog used here)

Reply:
Editable text-area is not good for sure.
I guess you're right about `make-info-dialog`. After rewrite it looks like:
```clojure
              (dialogs/make-info-dialog
                {:title "Delete Files?"
                 :icon :icon/circle-question
                 :header "Are you sure you want to delete these files?"
                 :content {:text (str "You are about to delete:\n" (->> selection
                                                    (map #(str "\u00A0\u00A0\u2022\u00A0"
                                                              (resource/resource-name %)))
                                                    (string/join "\n")))}
                 :buttons [{:text "Cancel"
                            :cancel-button true
                            :default-button true
                            :result false}
                           {:text "Delete"
                            :variant :danger
                            :result true}]}))
```
(I'm not sure what is better now: str or format in concatenating "You are about to delete" with flies list)
Another thing i want to clear is what do you mean by "wraps confirmation dialog". I didn't get this part. Is there something we can do with confirmation buttons/actions?

Reply:
Both `str` and `format` looks fine to me. 

By "it wraps confirmation dialog" I meant that `make-info-dialog` calls `make-confirmation-dialog` in its implementation. That was an argument for using `make-info-dialog` :)

Reply:
Good point. So, I just commit mentioned earlier code (it's using `make-info-dialog`) and that would be enough?

Reply:
Yes, your proposed code looks good to me 🤩

Reply:
Great, thank you!